CriticalMass/AppController.swift

```diff
@@ -6,6 +6,7 @@
 //
 
 import UIKit
+import Network
```

Comment:
I think this import statement isn't needed anymore

Reply:
right, fixed